tools/content/unimplemented_features_test.dart

```diff
@@ -0,0 +1,147 @@
+// Override `fluter test`'s default timeout
+@Timeout(Duration(minutes: 10))
+library;
+
+import 'dart:io';
+import 'dart:math';
+
+import 'package:checks/checks.dart';
+import 'package:html/dom.dart' as dom;
+import 'package:flutter/foundation.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:zulip/model/content.dart';
+
+import 'model.dart';
+
+
+/// Check if there are unimplemented features from the given corpuses of HTML
+/// contents from Zulip messages.
+///
+/// This test is meant to be run via `tools/content/check-features`.
+///
+/// To run it directly, use:
+///
+///   flutter test tools/content --dart-define=corpusDir=path/to/corpusDir
+///
+/// where `path/to/corpusDir` should be a directory containing files with
+/// outputs generated from tools/content/fetch_messages.dart.
+///
+/// Optionally, you can enable more details with `--dart-define=verbose=true`.
+///
+/// The test writes an overview of unimplemented features at the beginning to
+/// standard output, followed by the details of each feature.  To look for live
+/// examples, you can search on the Zulip community by message ID from all
+/// public channels.
+///
+/// For example, a search query like "near: 12345 channels: public" would work.
+///
+/// See also:
+/// * lib/model/content.dart, which implements of the content parser.
+/// * tools/content/fetch_messages.dart, which produces the corpuses.
+void main() async {
+  Future<void> checkForUnimplementedFeatureInFile(File file) async {
```

Comment:
nit:
```suggestion
  Future<void> checkForUnimplementedFeaturesInFile(File file) async {
```

It's checking for any possible unimplemented features, right? Vs. some particular unimplemented feature.